app/controllers/brexit_landing_page_controller.rb

```diff
@@ -1,12 +1,11 @@
 class BrexitLandingPageController < ApplicationController
-  include GovukPersonalisation::ControllerConcern
   include Slimmer::Headers
 
   skip_before_action :set_expiry
   before_action -> { set_expiry(1.minute) }
-  before_action -> { set_slimmer_headers(remove_search: true, show_accounts: logged_in? ? "signed-in" : "signed-out") }
+  before_action -> { set_slimmer_headers(remove_search: true) }
```

Comment:
This `remove_search` flag was useful back when the site header had a search field on the left hand side. There would be instances where we would want to hide the search field (e.g on the search page, or to put something else in there such as navigational links). It isn't useful here so I'd remove it.

Reply:
Ok cool so the entire L6 can go?

Reply:
Just force pushed!